### ISSUE TITLE
Add database

### ISSUE THREAD

**Issue by achul123** [PolarFox27/JobsMod#34]
It can be cool to add databases because imagine we have 2 server and we want to share the jobs xp ba we could with the database

**Comment by ELLIXspace** [comment]
I completely agree with this proposal. Adding a database will make it much easier to link or manage jobs across multiple worlds.

**Comment by achul123** [comment]
Add for 1.12.2
